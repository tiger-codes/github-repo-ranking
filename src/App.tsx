// import logo from './logo.svg';
import './App.css';
import Dashboard from './container/Dashboard';

function App() {
  return (
    <div className="App">
        <Dashboard />
    </div>
  );
}

export default App;
