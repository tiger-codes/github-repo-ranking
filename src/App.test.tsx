import * as renderer from 'react-test-renderer';
import App from './App';

describe('<App />', () => {
  it('renders correctly', () => {
    const AppComponent = renderer.create(<App />).toJSON();
    expect(AppComponent).toMatchSnapshot();
  });
});
