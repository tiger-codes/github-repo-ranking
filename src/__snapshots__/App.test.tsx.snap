// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<div
  className="App"
>
  <div
    className="dashboard-container"
  >
    <div
      className="search-panel"
    >
      <header>
        Git repositories by stars
      </header>
      <input
        onKeyPress={[Function]}
        placeholder="search"
        type="text"
      />
      <button
        onClick={[Function]}
      >
        search
      </button>
    </div>
  </div>
</div>
`;
